src/main/resources/static/api.yaml

```diff
@@ -3168,3 +3221,25 @@ components:
         - username
         - otp
         - sessionId
+    KeyDetail:
```

Comment:
are you going to return 1 or multiple? startdate seems to suggest that you have possibilities of multiple certs. In that case, we need to send an array of keys, right?

Reply:
It will be a single value. StartDate tells the system when it was introduced, If it has stored the public key value. It can delete it and fetch the new value

Reply:
It is a static value as of now